runtime/compiler/biome.vim

```diff
@@ -2,15 +2,21 @@
 " Compiler:     Biome (= linter for JavaScript, TypeScript, JSX, TSX, JSON,
 "               JSONC, HTML, Vue, Svelte, Astro, CSS, GraphQL and GritQL files)
 " Maintainer:   @Konfekt
-" Last Change:  2025 Nov 12
+" Last Change:  2025 Nov 16
 if exists("current_compiler") | finish | endif
 let current_compiler = "biome"
 
 let s:cpo_save = &cpo
 set cpo&vim
 
-exe 'CompilerSet makeprg=' .. escape('biome check --linter-enabled=true --formatter-enabled=false --assist-enabled=false --reporter=github '
-      \ .. get(b:, 'biome_makeprg_params', get(g:, 'biome_makeprg_params', '')), ' \|"')
+" CompilerSet makeprg=biome
+" CompilerSet makeprg=npx\ biome
+exe 'CompilerSet makeprg=' .. escape(
+                      \ (!empty(get(b:, 'biome_makeprg', get(g:, 'biome_makeprg', ''))) ?
+                      \   get(b:, 'biome_makeprg', get(g:, 'biome_makeprg', '')) :
```

Comment:
Biome is quite new, let's rather keep it simple and jut use the `javascript_node_makeprg` instead and get rid of the whole `biome_makeprg` option instead. This simplifies this whole thing here.